### PR TITLE
refactor(stories): migrate jsx to lit template

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "lint:js": "eslint \"**/*.js\"",
     "lint:yml": "eslint \"**/*.{yml,yaml}\"",
     "lint:styles": "stylelint \"**/*.scss\"",
-    "lint:lit": "yarn lit-analyzer \"src/components/**/*.ts\"",
```
Comment (jeripeierSBB) [inline package.json:40]: could we remove the react deps now?

**Review by jeripeierSBB — COMMENTED** [review]
Does `yarn generate:chromatic-stories` still work?

File: package.json (hunk 1 above)
Comment (jeripeierSBB) [reply package.json:40]: and jsx deps

Comment (TomMenga) [reply package.json:40]: I think yes.
We should also convert files from tsx to ts

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**